src/app/issues-viewer/label-chip-bar/label-chip-bar.component.ts

```diff
@@ -42,7 +42,9 @@ export class LabelChipBarComponent implements OnInit {
       (response) => {
         this.logger.debug('LabelChipBarComponent: Fetched labels from Github');
       },
-      (err) => {},
+      (err) => {
+        throw new Error('Failed to fetch labels from Github.');
+      },
```

Comment:
will this error be caught?
maybe it's better to invoke ErrorHandlingService here instead of throwing an error to the calling function above

Reply:
i agree, will change to invoking ErrorHandlingService here